src/DynamoCoreWpf/ViewModels/Core/DynamoViewModel.cs

```diff
@@ -528,6 +528,8 @@ protected void RegisterWatch3DViewModel(DefaultWatch3DViewModel watch3DViewModel
         {
             watch3DViewModel.Setup(this, factory);
             watch3DViewModels.Add(watch3DViewModel);
+            watch3DViewModel.Active = PreferenceSettings
+                .GetIsBackgroundPreviewActive(watch3DViewModel.PreferenceWatchName);
```

Comment:
hmm, maybe we can just use the typename, so this requires no changes on revit. thoughts?